go-controller/pkg/ovn/logical_switch_manager/logical_switch_manager.go

```diff
@@ -164,7 +165,7 @@ func (manager *LogicalSwitchManager) GetSwitchSubnets(switchName string) []*net.
 		}
 		return subnets
 	}
-	return nil
+	return make([]*net.IPNet, 0)
```

Comment:
why? I know this does not affect a user because it should be checking for len() anyway, but unless we want to make a distinction on nil vs empty array (that should be documented in the method godoc), the canonical thing would be to return nil, at least to me.

Reply:
Done.